core-web/src/main/java/org/silverpeas/core/webapi/socialnetwork/invitation/InvitationResource.java

```diff
@@ -71,6 +77,42 @@
     return asWebEntity(invitations, locatedAt(getUriInfo().getAbsolutePathBuilder()));
   }
 
+  @Path("{id}")
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  public InvitationEntity getInvitation(final @PathParam("id") Integer id) {
+    Invitation invitation = invitationService.getInvitation(id);
+    return asWebEntity(invitation, locatedAt(getUriInfo().getAbsolutePathBuilder()));
+  }
+
+  @DELETE
+  @Path("{id}")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response cancelInvitation(final @PathParam("id") Integer id) {
+    invitationService.ignoreInvitation(id);
+    return Response.ok().build();
+  }
+
+  @PUT
+  @Path("{id}")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response acceptInvitation(final @PathParam("id") Integer id) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)